roles/devscripts/tasks/135a_patch_ipv6_vips.yml

```diff
@@ -0,0 +1,56 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Fix dev-scripts bug where API_VIPS and INGRESS_VIPS use IPv4 DNS lookup even when IPv6 is configured
+# This patches the set_api_and_ingress_vip function in network.sh to use AAAA records
+# when IP_STACK is v6 or v6v4 (IPv6-only or IPv6-primary dual-stack deployments)
+- name: Fix API_VIPS IPv6 DNS lookup in dev-scripts
```

Comment:
This should be proposed as a fix to dev-scripts instead of patching it at runtime.

I do not see a patch proposed to https://github.com/openshift-metal3/dev-scripts ?

Reply:
I see the patching was removed?

Don't you need a Depends-On: https://github.com/openshift-metal3/dev-scripts/pull/1814 now ?